libiocage/lib/Jail.py

```diff
@@ -664,6 +665,50 @@ def _stop_vimage_network(self) -> None:
     def _configure_nameserver(self) -> None:
         self.config["resolver"].apply(self)
 
+    def _limit_resources(self) -> None:
+
+        for resource in self._resource_limit_config_keys:
+            try:
+                limit, action = self.config[resource].split(":", maxsplit=1)
+                command = [
+                    "/usr/bin/rctl",
+                    "-a",
+                    f"jail:{self.identifier}:{resource}:{action}={limit}"
+                ]
+                libiocage.lib.helpers.exec(command, logger=self.logger)
+            except KeyError:
+                pass
```

Comment:
i don't understand why we're not throwing, ir at least logging an error if a resource limit is (in our code) considered invalid

Reply:
updated issue #142 to cover this idea